packages/beacon-node/src/api/impl/validator/index.ts

```diff
@@ -938,6 +961,12 @@ export function getValidatorApi(
         // Requesting duties for next epoch is allow since they can be predicted with high probabilities.
         // @see `epochCtx.getBeaconProposersNextEpoch` JSDocs for rationale.
         indexes = state.epochCtx.getBeaconProposersNextEpoch();
+      } else if (epoch === stateEpoch - 1) {
+        const indexesPrevEpoch = state.epochCtx.getBeaconProposersPrevEpoch();
+        if (indexesPrevEpoch === null) {
```

Comment:
for a new state loaded from `Uint8Array` it's always be `null`, consider recomputing `indexesPrevEpoch` if it does not have instead

Reply:
This branch won't be executed if we load a state from `Uint8Array` because it will always load the state from request epoch which means it will call `getBeaconProposers`